contents/customers/researchgate.md

```diff
@@ -0,0 +1,65 @@
+
+“We grew up in a time when there weren’t a lot of third party services to solve the sorts of problems that arise from having so many publications and users,” says Paul McCloud, ResearchGate’s Head of Product Engineering. “Thankfully we’ve built a strong engineering team over the last 15 years and we’ve been able to build what we needed in-house.”
+
+Over the years ResearchGate’s team has built everything from internal experimentation frameworks and feature flagging systems to traffic experimentation systems and GraphQL clients. Many of these tools functioned to a high standard, but eventually the maintenance burden became too much. 
+
+“We built these tools well, but there comes a time when we can’t iterate of them as fast and we need to focus elsewhere,” says Paul. “This eventually became a strategic focus: How are we going to migrate more of our stack to third-party services so we can focus on our core competencies?”
+
+“And that’s what led us to PostHog.”
+
+### Scalability, support and on-site trips to say hello
+
+The scale of ResearchGate’s community meant that the decision to switch wasn’t made lightly, or quickly. Paul spoke with multiple providers before making a decision, with many conversations ending due to pricing and scalability issues. 
+
+“We have hundreds of millions of pageviews per month,” explains Paul. “So, based just on our volume, we’re always in that custom enterprise package and have to ‘Talk to Sales’. I had explicit conversations with [LaunchDarkly](/blog/posthog-vs-launchdarkly), [Optimizely](/blog/posthog-vs-optimizely), and others, but they were way out of our price range.”
+
+<BorderWrapper>
+<Quote
+    imageSource="/images/customers/paul-mccloud.jpg"
+    size="md"
+    name="Paul McCloud"
+    title="Head of Product Engineering, ResearchGate"
+    quote={`“Something I didn't get at the start was the clip at which PostHog adds new products. What you don't really understand until you've experienced it is that, because all these tools are built on the same fundamental architecture, the value of PostHog becomes exponential as new tools get connected!”`}
+/>
+</BorderWrapper>
+
+In addition to offering lower and more transparent pricing, PostHog also worked closely with Paul’s team to provide proactive support — including in an on-site visit earlier this year. 
+
+“If I email my Cloudflare support engineer it’ll be a few days before I get a reply and it’ll be a subpar response,” says Paul. “With PostHog though I particularly like that I get to deal with the relevant engineering expert on a topic. [Simon](/community/profiles/28895) will loop in [Neil](/community/profiles/28695), or [Marius](/community/profiles/30202), or [whoever](/people), and I don’t waste time playing the telephone game.”
```

Comment:
If you don't mind @joethreepwood , maybe we can make this generic rather than mentioning Cloudflare specifically, e.g. "If I email the support engineer from other providers we use..."